src/prasad/Identicalarray.java

```diff
@@ -0,0 +1,59 @@
+package prasad;
+/*WAP to find given 2 arrays are identical or not. 
+int[] arr1 = {10,12,55,32,17};
+int[] arr2 = {10,12,55,32,17}};
+output : Identical
+--------------------------------------------
+int[] arr1 = {10,12,55,32,17,99};
+int[] arr2 = {10,12,55,32,17}};
+output : Not identical
+------------------------------------
+int[] arr1 = {10,12,55,32,17};
+int[] arr2 = {10,12,99,32,17}};
+output : Not Identical*/
+
+public class Identicalarray {
+	boolean scanarray(int[]arraya,int[]arrayb) {
+			boolean flag=false;
+			if(arraya.length==arrayb.length) {
+				for(int index=0;index<arraya.length;index++) {
+					if(arraya[index]==arrayb[index])
+						flag=true;
+					else {
```

Comment:
you should put break in if condition, think here logically you want to break when arrays value is not matches. otherwise its fine & here i feel we need only if condition require.